src/app/api/stripe/webhooks/route.ts

```
@@ -0,0 +1,109 @@
+          data = event.data.object as Stripe.Checkout.Session;
+
+          if (!data.metadata?.userId) {
+            throw new Error("User ID is required!");
+          }
+
+          const user = await payload.findByID({
+            collection: "users",
+            id: data.metadata.userId,
+          });
+
+          if (!user) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)